src/lib/Containers/Gene.tsx

```diff
@@ -351,7 +347,8 @@ export default createRefetchContainer(
       ) {
       ...Header_gene
       ...About_gene
-      ...GeneArtworksGrid_gene
+      ...GeneArtworksGrid_gene @arguments(sort: $sort, medium: $medium, price_range: $price_range)
```

Comment:
This container didn't pass down the options

Reply:
Ahhhh, yes